portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -28,20 +28,23 @@
 
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet{
+
+    private final UserService userService = UserServiceFactory.getUserService();
+    private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    private static final String urlToRedirectToAfterUserLogsIn = "/nickname";
```

Comment:
for things that are constants, use `SCREMING_SNAKE_CASE`.

Reply:
Right, I forgot about this convention, thank you.